### ISSUE TITLE
eval_loss starts with nan

### ISSUE THREAD

**Issue by wasimmadha** [tuanh123789/Spark-TTS-finetune#1]
Hey, I have followed all the instruction, in-spite of that, I am getting eval_loss as nan

{'eval_loss': nan, 'eval_runtime': 7.6797, 'eval_samples_per_second': 85.289, 'eval_steps_per_second': 21.355, 'epoch': 0.01}

**Comment by tuanh123789** [comment]
Hi, what dataset do you use, language, how many hours

**Comment by tuanh123789** [comment]
![Image](https://github.com/user-attachments/assets/111d96bc-bd40-40f4-bd23-b54f309c38fe)
It's work fine with my language - Vietnamese

**Comment by wasimmadha** [comment]
I am using LJSpeech Data, mentioned in Readme

**Comment by OswaldoBornemann** [comment]
How do you generate the training label?

**Comment by manhcuong17072002** [comment]
@tuanh123789  Can you share more about the training process for Vietnamese data? For example: the dataset, the number of hours, whether multi-lingual training was conducted simultaneously, and the quality of the results. Thank you!

**Comment by tuanh123789** [comment]
> [@tuanh123789](https://github.com/tuanh123789) Can you share more about the training process for Vietnamese data? For example: the dataset, the number of hours, whether multi-lingual training was conducted simultaneously, and the quality of the results. Thank you!

I'll public pretrain for vietnamese and infomation soon

**Comment by tuanh123789** [comment]
> [@tuanh123789](https://github.com/tuanh123789) Can you share more about the training process for Vietnamese data? For example: the dataset, the number of hours, whether multi-lingual training was conducted simultaneously, and the quality of the results. Thank you!

https://huggingface.co/DragonLineageAI/Vi-SparkTTS-0.5B here is the vietnamese checkpoint

**Comment by manhcuong17072002** [comment]
@tuanh123789 thank you!
